tests/scripts/thread-cert/node.py

```diff
@@ -363,6 +365,46 @@ def dns_dig(self, server: str, name: str, qtype: str):
 
         return dig_result
 
+    def call_dbus_method(self, *args):
+        args = ' '.join(args)
+        return json.loads(
+            self.bash(f'python3 /app/third_party/openthread/repo/tests/scripts/thread-cert/call_dbus_method.py {args}')
+            [0])
+
+    def get_dbus_property(self, property_name):
+        return self.call_dbus_method('org.freedesktop.DBus.Properties', 'Get', 'io.openthread.BorderRouter',
+                                     property_name)
+
+    def get_border_routing_counters(self):
+        counters = self.get_dbus_property('BorderRoutingCounters')
```

Comment:
Is it intentionally adding the new dbus property verification here?

Typically node.py calls CLI command to get the information; for the new dbus property, may extend existing test_dbus_client.cpp in ot-br-posix when verifying?

Reply:
Yes I added this verification on purpose.

I decided to read the information from D-BUS instead of CLI because D-BUS is preferred in non-debug use cases. It's more important to ensure the correctness of D-BUS.

I didn't add the test to `test_dbus_client.cpp` because: 
- it's hard to send packets that can go through border routing in that test case. Hence we may not be able to fill in the counters with meaningful values and verify the values.
- On the other hand, we've already got some python based tests covering border routing cases. In such cases we can verify the counters in a more convenient way.